### ISSUE TITLE
[Bug Report] VTabs give an error when changing the router

### ISSUE THREAD

**Issue by FFxSquall** [vuetifyjs/vuetify#3581]
### Versions and Environment
**Vuetify:** 1.0.8
**Vue:** 2.5.16
**Browsers:** Chrome 65.0.3325.146
**OS:** Linux x86_64

### Steps to reproduce
1. yarn run dev
2. go to localhost:8080
3. after redirect click login

### Expected Behavior
console error (TypeError: Cannot read property 'clientWidth' of undefined) and visual tabs bug after return page

### Actual Behavior
no error

### Reproduction Link
[https://github.com/FFxSquall/vuetify_example](https://github.com/FFxSquall/vuetify_example)

### Other comments
The error occurs in `VTabs` because of a call through the` onContainerResize` timeout after an instance of tabs no longer exists

I use vuex and router. If the user refreshes the page or visits it, when it does not have token, the request to the server returns 401. The application redirects to the login page where there should not be a navigation drawer. Called `onContainerResize`, but the redirect already occurred and there are no tabs on the page

**Summaring**
When calling `onContainerResize`, used `setTimeout`, which does not guarantee the presence of a component or its parts in DOM

**Comment by FFxSquall** [comment]
I fixed some points and made a simpler example [(repo)](https://github.com/FFxSquall/vuetify_example/tree/simple_example). I do not understand how such a bug turns out, but I'll try to describe 2 case. 

**The First case (everything works correctly)**
1. Go to the login page http://localhost:8080/login (tabs have not yet been rendered even once)
2. Click on button
3. We see the correct `tabs-slider`

**The second case (does not work correctly)**
1. Go to the main page http://localhost:8080/
2. Tabs are rendered.
3. We throw out on the login page (emulate 401 status from the server)
4. Click the button
5. Redirect to the main page, but `tabs-slider` is displayed with the wrong left offset

**Comment by henon** [comment]
Happened to me too, when first loading the page of a SPA in release build. Here is the error from the console:

Uncaught TypeError: Cannot read property 'clientWidth' of undefined
    at setOverflow (vuetify.js:15790)
    at s.n [as setOverflow] (vue.esm.js:191)
    at s.callSlider (vuetify.js:15754)
    at n (vue.esm.js:191) 

I looked at the code and found it happens in v-tab component. I got several v-tabs in there so I don't know which one it was. Everything else the original poster observed seems to fit my error case too.
